metaflow/plugins/kfp/kfp.py

```diff
@@ -854,14 +902,34 @@ def build_kfp_dag(
                                 passed_in_split_indexes,
                                 preceding_kfp_component_op=next_kfp_component_op,
                                 preceding_component_outputs_dict=next_preceding_component_outputs_dict,
-                                volume_op=volume_op,
+                                workflow_uid=workflow_uid,
                             )
 
+            workflow_uid_op = None
+            if any(
```

Comment:
Guessing we want to remove this once we've upgraded KFP to a version that'll handle the `ownerReference` for us?

If so might be worthwhile adding in some comments here why this is needed.